src/Octeracts.ts

```diff
@@ -107,11 +109,21 @@ export class OcteractUpgrade extends DynamicUpgrade {
             freeLevelInfo = freeLevelInfo + '<span style="color: maroon"> (Softcapped) </span>'
         }
 
+        const isAffordable = costNextLevel <= player.wowOcteracts;
+        let affordTime = '';
+        if (!isMaxLevel && !isAffordable) {
+            const octPerSecond = octeractGainPerSecond();
```

Comment:
when could this be less than or equal to 0?

Reply:
When ascension score is too low (typically at the start of each Sing), Octeract production is 0.
I reused the 'Infinity' term that is used to show Oct production

Reply:
https://github.com/Pseudo-Corp/SynergismOfficial/blob/72b56eefad93e099ace295cc1d7a10bb2366cd11/src/Calculate.ts#L1438

1 / anything can never equal 0 in js, or am I mistaken?